pkg/app/api/grpcapi/web_api.go

```diff
@@ -1309,3 +1309,12 @@ func (a *WebAPI) ListAPIKeys(ctx context.Context, req *webservice.ListAPIKeysReq
 		Keys: apiKeys,
 	}, nil
 }
+
+// GetApplicationDeploymentFrequency [WIP] Made a temporary implementation to satisfy interface
+func (a *WebAPI) GetApplicationDeploymentFrequency(ctx context.Context, req *webservice.GetApplicationDeploymentFrequencyRequest) (*webservice.GetApplicationDeploymentFrequencyResponse, error) {
```

Comment:
With `_`, can be a clear indication of what not to use in a function.
```suggestion
func (a *WebAPI) GetApplicationDeploymentFrequency(_ context.Context, _ *webservice.GetApplicationDeploymentFrequencyRequest) (*webservice.GetApplicationDeploymentFrequencyResponse, error) {
```